README.md

```diff
@@ -1,43 +1,52 @@
-# InvisibleServant
+# UnseenServant
 
 A D&D game organisation tool for discord
 
 ## Development install
 
 - Clone repository
-  - `git clone https://github.com/OperationXen/InvisibleServant.git`
+
+  - `git clone https://github.com/OperationXen/UnseenServant.git`
+
 - Create and activate python virtual environment (optional)
 
   - `python -m pip install virtualenv`
   - `python -m virtualenv venv`
   - `./venv/script/activate`
-  - Install dependencies
+
+- Install dependencies
+
+  - Ensure C++ build tools are installed: https://visualstudio.microsoft.com/visual-cpp-build-tools
+  - `pip install setuptools` <- can this go in requirements.txt?
```

Comment:
@OperationXen not sure if I can just add this to the requirements.txt? it was needed by the newer version of python